debug_src/debug.js

```diff
@@ -0,0 +1,78 @@
+const preact = require('../');
+
+const original = {
+	h: preact.h
+};
+
+const config = {
+	undefinedComponents: true,
+	stringRefs: true,
+	arrayChildren: true
+};
+
+preact.h = function(nodeName, attributes) {
```

Comment:
Stacks aren't that simple. Firefox have weird stack format and you cannot just modify it. There is no docs about the format and people in Mozilla IRC doesn't answer really to that question. There might be some lib by Fx extensions team, but I don't know about it.